acceptance/acceptance_test.go

```diff
@@ -30,11 +31,16 @@ var KeepTmp bool
 // example: var SingleTest = "bundle/variables/empty"
 var SingleTest = ""
 
+// TODO: Add a custom diff function based on the method and path.
```

Comment:
Personally I don't think it's going to be needed. I might be wrong - we can discuss and decide otherwise but let's not put such notes there.

Reply:
Yeah, I agree. Not needed yet. Maybe in the future.